src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -80,7 +80,7 @@ public void Initialize(ExtensionConfigContext context)
             IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
             extensions.RegisterBindingRules<DocumentDBAttribute>(ValidateConnection, nameResolver, outputProvider, clientProvider, jArrayProvider, enumerableProvider, inputProvider);
 
-            context.Config.RegisterBindingExtensions(new CosmosDBTriggerAttributeBindingProvider(ResolveConnectionString(null), ResolveConnectionString(null), LeaseOptions));
+            context.Config.RegisterBindingExtensions(new CosmosDBTriggerAttributeBindingProvider(nameResolver, ResolveConnectionString(null), ResolveConnectionString(null), LeaseOptions));
```

Comment:
Makes sense, I didn't fully understand the nameResolver back when I created this 😄